libbeat/docs/shared-docker.asciidoc

```diff
@@ -41,26 +41,183 @@ https://www.docker.elastic.co[www.docker.elastic.co].
 endif::[]
 
 [float]
+==== Run the Beat setup
+
+Running the Beat with the setup command will create the index pattern, load visualizations, dashboards, and machine learning jobs.  Run this command:
+
+ifeval::[("{beatname_lc}"=="filebeat") or ("{beatname_lc}"=="metricbeat") or ("{beatname_lc}"=="heartbeat")]
+["source", "sh", subs="attributes"]
+--------------------------------------------
+docker run \
+{dockerimage} \
+setup -E setup.kibana.host=kibana:5601 \
+-E output.elasticsearch.hosts=["elasticsearch:9200"]
+--------------------------------------------
+endif::[]
+
+ifeval::["{beatname_lc}"=="packetbeat"]
+["source", "sh", subs="attributes"]
+--------------------------------------------
+docker run \
+--cap-add=NET_ADMIN \
+{dockerimage} \
+setup -E setup.kibana.host=kibana:5601 \
+-E output.elasticsearch.hosts=["elasticsearch:9200"]
+--------------------------------------------
+endif::[]
+
+ifeval::["{beatname_lc}"=="auditbeat"]
+["source", "sh", subs="attributes"]
+--------------------------------------------
+docker run \
+  --cap-add="AUDIT_CONTROL" \
+  --cap-add="AUDIT_READ" \
+  {dockerimage} \
+  setup -E setup.kibana.host=kibana:5601 \
+  -E output.elasticsearch.hosts=["elasticsearch:9200"]
+--------------------------------------------
+endif::[]
+
+NOTE: Substitute your Kibana and Elasticsearch hosts and ports above, and if you are using the hosted Elasticsearch Service in Elastic Cloud, replace the `-E output.elasticsearch.hosts` line above with the Cloud ID and elastic password using this syntax:
+
+--------------------------------------------
+-E cloud.id=<Cloud ID from Elasticsearch Service> \
+-E cloud.auth=elastic:<elastic password>
+--------------------------------------------
+
 ==== Configure {beatname_uc} on Docker
 
 The Docker image provides several methods for configuring {beatname_uc}. The
-conventional approach is to provide a configuration file via a bind mount, but 
+conventional approach is to provide a configuration file via a volume mount, but 
 it's also possible to create a custom image with your
 configuration included.
 
 [float]
-===== Bind-mounted configuration
+===== The configuration file
 
-One way to configure {beatname_uc} on Docker is to provide +{beatname_lc}.yml+ via a bind mount.
-With +docker run+, the bind mount can be specified like this:
+Download this file as a starting point:
```

Comment:
we could state it is an example configuration?

Reply:
will do.